lib/nostrum/voice/opus.ex

```diff
@@ -147,35 +130,37 @@ defmodule Nostrum.Voice.Opus do
         } = _state
       )
       when is_list(body) do
+    segment_table_raw =
+      body
+      |> Enum.map(&byte_size(&1))
```

Comment:
```suggestion
      |> Enum.map(&byte_size/1)
```

My BEAM-fu tells me this avoids an extra anonymous function, so there is space for an insanely important optimization here, saving us lightyears of CPU time.